chain/network-primitives/src/network_protocol/mod.rs

```diff
@@ -80,8 +80,8 @@ impl FromStr for PeerInfo {
             addr = None;
             account_id = None;
         } else if chunks.len() == 2 {
-            if let Ok(x) = chunks[1].parse::<SocketAddr>() {
```

Comment:
while you are at it, would you mind adding a docstring to this function listing accepted formats?

Reply:
also the following docstrings require an update:
- https://github.com/near/nearcore/blob/c39ed74feeeb43e27367e4a90f47540b5d361d59/nearcore/src/config.rs#L212
- (this one is missing the formats in the docstring, please add): https://github.com/near/nearcore/blob/c39ed74feeeb43e27367e4a90f47540b5d361d59/nearcore/src/config.rs#L206